catalyst_voices/packages/libs/catalyst_key_derivation/rust/src/lib.rs

```diff
@@ -1,2 +1,20 @@
+//! Catalyst Key Derivation
+
+// cspell: words uninlined
+
 pub mod api;
+#[allow(
```

Comment:
Just a question, this disabled lints for the autogenerated code right ?

Reply:
Yep, without this it won't pass the CI.